instructions/steps.md

```diff
@@ -577,7 +614,7 @@ We could write
 
 This is great!
 Dafny now believes us that `components[5]` will _always_ be valid.
-You can even ask Dafny `assert |components| < 7;` and it will object.
+You can even ask Dafny `assert |components| < 7;` and it will not object.
```

Comment:
not? `7 != 6` So I think that Dafny will object, because it MUST be 6

Reply:
Oh did you mean `assert |components| == 7;`?

Reply:
OH, wait! I see
```suggestion
You can even ask Dafny `assert 7 < |components|;` and it will object.
```